### PR TITLE
docs: create ADR for plugin decision

### PR DESCRIPTION
**Pull request by bseverino**
**JIRA:**

[MST-852](https://openedx.atlassian.net/browse/MST-852)

**Pre-Merge Checklist:**

- [x] Confirmed Github reports all automated tests/checks are passing.
- [x] Approved by at least one additional reviewer.

### CODE REVIEW

File: docs/decisions/0002-name-affirmation-plugin.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+2. Proctoring Email Template Overrides
+--------------------------------------
+
+Status
+------
+
+Accepted
+
+Context
```
Comment (Zacharis278) [inline docs/decisions/0002-name-affirmation-plugin.rst:9]: I think we should give some background on why this was created as a plugin as opposed to other options (IDA, platform app)

File: docs/decisions/0002-name-affirmation-plugin.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
+2. Proctoring Email Template Overrides
```
Comment (Zacharis278) [inline docs/decisions/0002-name-affirmation-plugin.rst:1]:
```suggestion
2. Name Affirmation Plugin
```
copypasta

File: docs/decisions/0002-name-affirmation-plugin.rst (hunk 1 above)
Comment (schenedx) [reply docs/decisions/0002-name-affirmation-plugin.rst:9]: Agree. Please discuss the product context. Please answer the question: why we need this plugin at all?

File: docs/decisions/0002-name-affirmation-plugin.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Accepted
+
+Context
+-------
+
+`edx-name-affirmation` is intended to be an optional feature for the open edX community. As such, we
+should allow the consuming application (in our case, `edx-platform <https://github.com/edx/edx-platform>`_)
+to determine whether or not this feature should be installed.
+
+Decision
```
Comment (schenedx) [inline docs/decisions/0002-name-affirmation-plugin.rst:16]: Could we place have a section called `options`. Please list out all the different design options you hashed out. See the list of options you documented at https://openedx.atlassian.net/wiki/spaces/PT/pages/2594570529/Discovery+Name+Affirmation+Technical+Design

**Review by schenedx — CHANGES_REQUESTED** [review]
See inline comments

File: docs/decisions/0002-name-affirmation-plugin.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+should allow the consuming application (in our case, `edx-platform <https://github.com/edx/edx-platform>`_)
+to determine whether or not this feature should be installed.
+
+Options
+-------
+
+**edx-platform app**
+
+- Create a new application directly in `edx-platform`.
+- This is the easiest option, but doesn't fit our purposes in order to decouple the code from platform.
```
Comment (ashultz0) [inline docs/decisions/0002-name-affirmation-plugin.rst:22]: maybe change to "but couples the code to platform"

**Review by schenedx — APPROVED** [review]
👍

**Pull request merged**